### PR TITLE
Contrib astropy

### PR DESCRIPTION
**Pull request by tloizou**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/da8f28226b42e16a6111 for a full test report.

File: easybuild/easyconfigs/s/setuptools/setuptools-18.4-goolf-1.4.10-Python-2.7.3.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+dependencies = [(python, pythonversion)]
+
+py_short_ver = ".".join(pythonversion.split(".")[0:2])
+pylibdir = "lib/python%s/site-packages/%s" % (py_short_ver, name)
+
+sanity_check_paths = {
+                      'files': ["bin/easy_install", "%s-%s-py%s.egg" % (pylibdir, version, py_short_ver)],
+                      'dirs': []
+                     }
```
Comment (boegel) [inline easybuild/easyconfigs/s/setuptools/setuptools-18.4-goolf-1.4.10-Python-2.7.3.eb]: @tloizou please fix indent here?

```python
sanity_check_paths = {
    'files': ["bin/easy_install", "%s-%s-py%s.egg" % (pylibdir, version, py_short_ver)],
    'dirs': [],
}
```

Comment (tloizou) [inline easybuild/easyconfigs/s/setuptools/setuptools-18.4-goolf-1.4.10-Python-2.7.3.eb]: @boegel: Indent fixed!

**Comment by boegel** [comment]
Going in, thanks @tloizou!

**Pull request merged**